dev/run-pip-tests

```diff
@@ -76,8 +76,12 @@ for python in "${PYTHON_EXECS[@]}"; do
     VIRTUALENV_PATH="$VIRTUALENV_BASE"/$python
     rm -rf "$VIRTUALENV_PATH"
     if [ -n "$USE_CONDA" ]; then
+      if [ -f "$CONDA_PREFIX/etc/profile.d/conda.sh" ]; then
+        # See also https://github.com/conda/conda/issues/7980
+        source $CONDA_PREFIX/etc/profile.d/conda.sh
+      fi
       conda create -y -p "$VIRTUALENV_PATH" python=$python numpy pandas pip setuptools
-      source activate "$VIRTUALENV_PATH"
+      conda activate "$VIRTUALENV_PATH"
```

Comment:
`conda activate` seems the recommended way, https://docs.conda.io/projects/conda/en/latest/release-notes.html#id203
> With conda 4.4, conda activate and conda deactivate are now the preferred commands for activating and deactivating environments.